File: C/crescente.c
#include <stdio.h>

int main() {
    
    int numero1, numero2;
    
    while (1) {
        printf("Digite dois numeros: ");
        scanf("%d", &numero1);
        scanf("%d", &numero2);
        
        
        if (numero1 == numero2) {
            printf("Valores iguais encerra o programa");
        }
        
         if (numero1 < numero2) {
            printf("Ordem crescente\n");
        } else {
            printf("Ordem decrescente\n");
        }
    
        
    }

    return 0;
}

File: JAVA/crescente.java
import java.util.Locale;
import java.util.Scanner;

public class crescente {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int primeiroValor, segundoValor;
		
		while (true) {
			System.out.println("Digite dois numeros: ");
			primeiroValor = sc.nextInt();
			segundoValor = sc.nextInt();
			
			if (primeiroValor == segundoValor) {
				System.out.println("Encerrar programa");
				break;
	
			} else if (primeiroValor < segundoValor) {
				System.out.println("CRESCENTE!");
			} else {
				System.out.println("DECRESCENTE");
			}
		}
		
		sc.close();
	}

}
